ingress/dashboard_ingress_setup.md

```diff
@@ -0,0 +1,216 @@
+---
```

Comment:
This is similar to the one from the handbook, right? Do we plan on removing that from there?

Reply:
this one is modified to setup ingress for dashboards mostly, that states how to install KDCC, that knowledge can be generally available as well since it's one of our major feature.